register/emails.py

```diff
@@ -17,3 +21,52 @@ def sendgrid_send(recipients, subject, substitutions, template_id):
     # Replace substitutions in sendgrid template
     mail.substitutions = substitutions
     mail.send()
+
+
+class MailListManager:
+    WINTER_17_LIST_ID = "876178"
+
+    def __init__(self):
+        self.sg = sendgrid.SendGridAPIClient(apikey=SENDGRID_API_KEY)
+
+    def add_recipient_to_list(self, recipient_id, list_id):
```

Comment:
Is this used outside of the class? If it's not then add _ before the name (python convention for private functions) `_add_recipient_to_list`